manifest.php

```diff
@@ -28,7 +28,7 @@
   'label'       => 'Delivery Management',
   'description' => 'Manages deliveries using the ontology',
   'license'     => 'GPL-2.0',
-  'version'     => '3.22.0',
+  'version'     => '3.23.0',
 	'author'      => 'Open Assessment Technologies SA',
 	'requires'    => array(
         'generis'     => '>=5.11.0',
```

Comment:
depends on tao and generis (new version)

Reply:
In this case a downgrade it's necessary?

Reply:
of course not, I mixed generis with tao, sorry